src/sentry/static/sentry/app/components/sidebar/sidebarMenuItem.jsx

```diff
@@ -48,32 +48,23 @@ const MenuItemLabel = styled('span')`
         `};
 `;
 
-const MenuItemLink = styled(({to, href, external, ...props}) => {
-  if (to) {
-    return <Link to={to} href={href} {...props} />;
-  }
-
-  if (href) {
-    const Component = external ? ExternalLink : Link;
-    return <Component href={href} {...props} />;
-  }
-
-  return <div {...props} />;
-})`
-  color: ${p => p.theme.gray5};
+const getMenuItemStyles = p => css`
```

Comment:
Can we remove the `css`, it can just return a string (this will be what we want to do in emotion@10)

Reply:
Talked to @billyvg offline about this, should be good as long as we compose it.